scripts/events_module/condition_events.py

```diff
@@ -678,11 +678,11 @@ def handle_already_disabled(cat):
             status = cat.moon_skip_permanent_condition(condition)
 
             # if cat is dead, break
-            if cat.dead or cat.status == 'leader':
+            if cat.dead:
```

Comment:
If the leader has more than one live left, this if statement will not pas.
Maybe you can add a variable to check how many lives are left before the function call "moon_skip_permanent_condition" and check if the number decreases?

Reply:
Did you mean something like what I changed now? Sorry if I got it wrong, I'm basically learning from scratch through deduction/trial and error!